texastribune/scuole
# Askted auto update

**To test:**

`make local/db-fetch`
`make data/bootstrap-test-askted`

Then check to see if campuses and districts have principals and superintendents. 

**Next steps:**

Set up to run automatically every day and load into production db
